api/src/main/java/com/intern/api/ActuatorServiceApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.intern.api;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class ActuatorServiceApplication {
```

Comment:
Entry point of each module usually are called as the module itself. So for example for the `api` module the class will be called `ApiApplication`, for `storage` - `StorageApplication`.